src/unitxt/operators.py

```diff
@@ -1475,7 +1532,7 @@ class AddFieldNamePrefix(StreamInstanceOperator):
     """Adds a prefix to each field name in each instance of a stream.
 
     Args:
-        prefix_dict (Dict[str, str]): A dictionary mapping stream names to prefixes.
```

Comment:
Operator `AddFieldNamePrefix` is used nowhere. Shall I remove it?

Reply:
yes looks unnecessary

Reply:
Removed.. as part of this branch. If this branch will not be merged into main, I will remember to remove `AddFieldNamePrefix` in another PR..